### PR TITLE
show bandwidth stats

### PR DESCRIPTION
**Pull request by jensengar**

### CODE REVIEW

File: addon/components/webrtc-troubleshooter/template.hbs [hunk 1]
```diff
@@ -1,99 +1,119 @@
...
+        {{else if checkBandwidthSuccess}}
+          {{t "webrtcTroubleshoot.checkBandwidthSuccess"}} <i class="status-icon pc pc-check"></i>
+        {{else}}
+          {{t "webrtcTroubleshoot.checkBandwidthFail"}} <i class="status-icon pc pc-close"></i>
+        {{/if}}
+      </div>
+    </div>
+
+    <div class="bandwidth-stats">
+      {{#if bandwidthStats}}
```
Comment (xdumaine) [inline addon/components/webrtc-troubleshooter/template.hbs]: Should this `if` block move outside the parent div, so we don't render an empty div when it's disabled?

File: app/translations/webrtc-troubleshoot/en-us.json [hunk 2]
```diff
@@ -19,6 +19,18 @@
...
     "checkConnectivity": "Checking connectivity ...",
     "checkConnectivitySuccess": "No connectivity issues",
-    "checkConnectivityFail": "Oops, there was a connectivity issue!"
+    "checkConnectivityFail": "Oops, there was a connectivity issue!",
+    "showBandwidthStats": "Show Bandwidth Stats",
+    "hideBandwidthStats": "Hide Bandwidth Stats",
+    "resolution": "Resolution: {value}",
+    "bpsAvg": "Bit Rate Average: {value, number}",
+    "bpsMax": "Bit Rate Max: {value, number}",
+    "lostPackets": "Lost Packets: {value, number}",
```
Comment (xdumaine) [inline app/translations/webrtc-troubleshoot/en-us.json]: Lets do Packet Loss as a percent, instead of the `lost` number count.

**Comment by xdumaine** [comment]
@jensengar You need to bump the dependency for the webrtc-troubleshooter to 2.x, right?

**Comment by jensengar** [comment]
Yes, I already bumped the dep.

**Pull request merged**